controller.py

```diff
@@ -159,16 +189,32 @@ def remove_items(self):
                     )
             if message:
                 messages.append(message)
+        self._db.commit()
```

Comment:
The database session should already be committed at the end of each request by `shutdown_session` in `app.py`, so this shouldn't be necessary. If it is, then something's wrong with `app.py.`